The benchmark called "Benchmark" fails when executed because nested defs aren't enabled.

I tried running the benchmark "Benchmark" via the following commands:

```
$ go get go.starlark.net/starlark
$ go test -bench="Benchmark$" go.starlark.net/starlark
```

and got the following output

```
--- FAIL: Benchmark
    eval_test.go:445: /home/mknyszek/go/src/go.starlark.net/starlark/testdata/benchmark.star:9:3: this Starlark dialect does not support nested def
FAIL
exit status 1
FAIL    go.starlark.net/starlark        0.185s
```

Perhaps I missed a flag or something somewhere, so I apologize in advance for the noise if so.

On a side note, I'm personally interested in seeing some end-to-end benchmarks of more complex and realistic Starlark scripts. :)

--- a/starlark/bench_test.go
+++ b/starlark/bench_test.go
@@ -16,16 +16,26 @@ import (
 )
 
 func Benchmark(b *testing.B) {
+	defer setOptions("")
+
 	testdata := starlarktest.DataFile("starlark", ".")
 	thread := new(starlark.Thread)
 	for _, file := range []string{
 		"testdata/benchmark.star",
 		// ...
 	} {
+
 		filename := filepath.Join(testdata, file)
 
+		src, err := ioutil.ReadFile(filename)
+		if err != nil {
+			b.Error(err)
+			continue
+		}
+		setOptions(string(src))
+
 		// Evaluate the file once.
-		globals, err := starlark.ExecFile(thread, filename, nil, nil)
+		globals, err := starlark.ExecFile(thread, filename, src, nil)
 		if err != nil {
 			reportEvalError(b, err)
 		}

--- a/starlark/testdata/benchmark.star
+++ b/starlark/testdata/benchmark.star
@@ -1,22 +1,27 @@
 # Benchmarks of Starlark execution
+# option:nesteddef
 
 def bench_range():
-  return range(200)
+    return range(200)
 
 # Make a 2-level call tree of 100 * 100 calls.
 def bench_calling():
-  list = range(100)
-  def g():
-    for x in list:
-      pass
-  def f():
-    for x in list:
-      g()
-  f()
+    list = range(100)
+
+    def g():
+        for x in list:
+            pass
+
+    def f():
+        for x in list:
+            g()
+
+    f()
 
 # Measure overhead of calling a trivial built-in method.
 emptydict = {}
 range1000 = range(1000)
+
 def bench_builtin_method():
-  for _ in range1000:
-    emptydict.get(None)
+    for _ in range1000:
+        emptydict.get(None)

